### ISSUE TITLE
v2 weight not released ?

### ISSUE THREAD

**Issue by Samuel-gwb** [bytedance/Protenix#295]
Error happened when downloading protenix-v2.pt. 
The new version weight is not available, is it?

**Comment by zhangyuxuann** [comment]
@Samuel-gwb Thank you for reporting this issue. Please note that the accessibility of the protenix-v2 checkpoint is currently under review as part of our company-level internal evaluation process. We are unable to provide a specific timeline at this stage, but we will update this thread once the process is complete and we have further information. We appreciate your understanding and patience.

**Comment by Samuel-gwb** [comment]
OK, thanks 
Also congratulations for your great work !

**Comment by bschussheim-lila** [comment]
@zhangyuxuann just to confirm, the design capabilities will not be open sourced but the structure prediction model weights will be released, right?

**Comment by liangxu011** [comment]
@zhangyuxuann Thank you for the great work! Can I ask:
- Is there a chance the v2 weights won't be released?
- Is there a plan to release a v2 model with a later training date cutoff?
- Is there a plan to release a v2 model with constraints?

**Comment by GriffinYoung** [comment]
Also wondering if there's an update about the v2 weights?

**Comment by bschuss02** [comment]
@zhangyuxuann could we please have an update on the status of the v2 weights? It's been a while

**Comment by rjrich** [comment]
I do not understand why Protenix v.2 was announced as released when the corresponding weights have not been released. So, apparently, it is possible to install v.2, but it appears that running inferences with this version and interpreting the results would be scientifically and technically problematic. Should current and potential users of Protenix revert to v.1?

**Comment by astronomer91** [comment]
just wondering if you know if the v2 model weights have been released yet?
